source/cloud/aws/eks.md

```diff
@@ -1,89 +1,117 @@
+## Create the Kubernetes cluster
+
+Now we can launch a GPU enabled EKS cluster. First launch an EKS cluster with `eksctl`.
+
+```console
+$ eksctl create cluster rapids \
+                      --version 1.24 \
+                      --nodes 3 \
+                      --node-type=p3.8xlarge \
+                      --timeout=40m \
+                      --ssh-access \
+                      --ssh-public-key <public key ID> \  # Be sure to set your public key ID here
+                      --region us-east-1 \
+                      --zones=us-east-1c,us-east-1b,us-east-1d \
+                      --auto-kubeconfig \
+                      --install-nvidia-plugin=false
+```
 
-Update the path to the ssh-public-key to point to the folder and file where your public key is saved.
+With this command, you’ve launched an EKS cluster called `rapids`. You’ve specified that it should use nodes of type `p3.8xlarge`. We also specified that we don't want to install the NVIDIA drivers as we will do that with the NVIDIA operator.
 
-**4. Install GPU addon:**
+To access the cluster we need to pull down the credentials.
 
-```shell
-$ kubectl create -f https://raw.githubusercontent.com/NVIDIA/k8s-device-plugin/1.0.0-beta4/nvidia-device-plugin.yml
+```console
+$ aws eks --region us-east-1 update-kubeconfig --name rapids
 ```
 
-**5. Install RAPIDS helm repo:**
+## Install drivers
+
+Next, [install the NVIDIA drivers](https://docs.nvidia.com/datacenter/cloud-native/gpu-operator/getting-started.html) onto each node.
 
-```shell
-$ helm repo add rapidsai https://helm.rapids.ai
-$ helm repo update
+```console
+$ helm install --repo https://helm.ngc.nvidia.com/nvidia --wait --generate-name -n gpu-operator --create-namespace gpu-operator
+NAME: gpu-operator-1670843572
+NAMESPACE: gpu-operator
+STATUS: deployed
+REVISION: 1
+TEST SUITE: None
 ```
 
-**6. Install helm chart:**
+Verify that the NVIDIA drivers are successfully installed.
 
-```shell
-$ helm install --set dask.scheduler.serviceType="LoadBalancer" --set dask.jupyter.serviceType="LoadBalancer" rapidstest rapidsai/rapidsai
+```console
+$ kubectl get po -A --watch | grep nvidia
+kube-system   nvidia-driver-installer-6zwcn                                 1/1     Running   0         8m47s
+kube-system   nvidia-driver-installer-8zmmn                                 1/1     Running   0         8m47s
+kube-system   nvidia-driver-installer-mjkb8                                 1/1     Running   0         8m47s
+kube-system   nvidia-gpu-device-plugin-5ffkm                                1/1     Running   0         13m
+kube-system   nvidia-gpu-device-plugin-d599s                                1/1     Running   0         13m
+kube-system   nvidia-gpu-device-plugin-jrgjh                                1/1     Running   0         13m
 ```
 
-**7. Accessing your cluster:**
+After your drivers are installed, create a quick sample pod that uses some GPU compute to make sure that everything is working as expected.
```

Comment:
Looks like this quick test is common among the managed Kubernetes services and could be made an `_includes` in a follow up PR

Reply:
Oh nice idea! I'll open an issue to track.